Client/Assets/Themes/ACE618A3-D6FC-45A4-94F2-1793C40AE927.json

```diff
@@ -27,7 +27,7 @@
         "stats": {
             "ads": "0xFB542B",
             "trackers": "0x742BC4",
-            "httpse": "0xA0A5EB",
+            "dataSaved": "0xA0A5EB",
             "timeSaved": "0xffffff"
```

Comment:
Are we sure we want to separate these? Will potential iOS 14 widgets or detailed stats screens want to show a HTTPSE stat?

Reply:
No, not from JSON right now: 

https://github.com/brave/brave-ios/commit/014029b529ba199890799792c43f68fc57564f1f#diff-0f101c2a123f5e274262466a7c30ad4789da612782cf696ec5e1fa0c05066f8cR24-R35

But cool, if we're not planning on bringing back HTTPSE ever again we can maybe also remove HTTPSE tracking code as well (and HTTPSE lib)

Reply:
Talked to James, no plans to show the stat.
But the httpse detection code is still used to show one educational popup, i think we have to keep it for now